.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  semi: true,
+  singleQuote: true,
+  tabWidth: 2,
+  printWidth: 80,
+};
```

Comment:
What is the motivation for these values? Ideally in my opinion this should be left blank (`{}`) as the prettier defaults should be all we need.